Default scan active (#11788)
nrfconnect/sdk-nrf: samples/wifi/scan/README.rst

********

This sample can perform Wi-Fi scan operations in the 2.4GHz and 5GHz bands.

> The default scan type is active scan. However, you can configure the scan type to be either active or passive.

Reviewer: ```suggestion
The default scan type is active scan.
However, you can configure the scan type to be either active or passive.
```

********

This sample can perform Wi-Fi scan operations in the 2.4GHz and 5GHz bands.
The default scan type is active scan.
However, you can configure the scan type to be either active or passive.

Using this sample, the development kit can scan for available access points in :abbr:`STA (Station)` mode.

> 

Reviewer: You might want to add here the Configuration heading. See the sample doc template: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/sample_README.html#configuration

Author: Yeah, added, please review

Reviewer: Not quite yet there. Proposing to have:

Configuration
***************
|config|

note about the scan type

Configuration options
==============

I believe :kconfig:option:`CONFIG_WIFI_SCAN_TYPE_SELECT` should work here, as the new option is going to be available in the Kconfig reference, right?

Reviewer: The parsing went a bit wrong in my comment, but I hope you get the idea. Just like in the template.

Reviewer: Sorry for stealing the review from you @kittydepa. Please continue from here.

********

This sample can perform Wi-Fi scan operations in the 2.4GHz and 5GHz bands.

> Default scan type is active scan. Optionaly user can select scan type (active/passive).

Reviewer 2: ```suggestion
The default scan type is active scan. However, you can configure the scan type to be either active or passive.
```

Using this sample, the development kit can scan for available access points in :abbr:`STA (Station)` mode.

|config|

You may configure the Wi-Fi scan type in the :file:`prj.conf` file:

> * Scan type

Reviewer: Do we need this line?

Author: No removed this line, as refernce in prj.conf is no more. Updated doc accordingly.

.. note::

>        You can also use ``menuconfig`` to enable ``Scan type`` option.

Reviewer 2: ```suggestion
       You can also use ``menuconfig`` to enable the ``Scan type`` option.
```

********

This sample can perform Wi-Fi scan operations in the 2.4GHz and 5GHz bands.
The default scan type is active scan.
However, you can configure the scan type to be either active or passive.

Using this sample, the development kit can scan for available access points in :abbr:`STA (Station)` mode.

Configuration
*************

|config|

> Select the scan type to be used by the sample:

Reviewer: ```suggestion
Select the scan type to be used by the sample.
```
Alternatively:
Now, if there are no other important configuration options, you could dump the Configuration options section and just have it in this Configuration section.
Something like: "Use the :kconfig:option:`CONFIG_WIFI_SCAN_TYPE_SELECT` Kconfig option to set the scan type to be used by the sample."
